### DISCUSSION TITLE
Расписание зачетной, экзаменационной сессий

### DISCUSSION THREAD

**Discussion by ValeryVerkhoturov** [0niel/university-app#211] [Идеи]
**Проблема**
В приложении невозможно посмотреть расписание зачетной, экзаменационной сессий.

**Решение**
Добавить разделы зачетной, экзаменационной сессий.

**Альтернативы**
«Сезонное» появление разделов зачетной, экзаменационной сессий.

**Comment by Dragonprod** [comment]
Реализовано
